### ISSUE TITLE
TransformPropTypes deprecated since RN 0.58

### ISSUE THREAD

**Issue by mxmzb** [tipsi/tipsi-stripe#430]
The module `react-native/Libraries/StyleSheet/StyleSheetPropType` has been dropped in RN 0.58, see:

https://github.com/facebook/react-native/blob/v0.57.8/Libraries/StyleSheet/TransformPropTypes.js
vs
https://github.com/facebook/react-native/tree/v0.58.1/Libraries/StyleSheet

It's currently being called from here:
https://github.com/tipsi/tipsi-stripe/blob/339cea10bc7c45b341154c37d3ef4d96db6cc906/src/components/PaymentCardTextField.js#L12

and will lead to the following error:
```
error: bundling failed: Error: Unable to resolve module `react-native/Libraries/StyleSheet/StyleSheetPropType` from `yourapp/node_modules/tipsi-stripe/src/components/PaymentCardTextField.js`: Module `react-native/Libraries/StyleSheet/StyleSheetPropType` does not exist in the Haste module map
```

**Comment by cihati** [comment]
@cybergrind is there a chance to fix this issue? @philipgraf already kindly created https://github.com/tipsi/tipsi-stripe/pull/431 . Many thanks!

**Comment by cybergrind** [comment]
@cihati merged and uploaded as 7.2.0
